modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java

```diff
@@ -180,6 +181,37 @@ public ObjectEntryDisplayContextImpl(
 			WebKeys.THEME_DISPLAY);
 	}
 
+	public String getAPIURL() throws PortalException {
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.fetchObjectRelationship(
+				ParamUtil.getLong(
+					_objectRequestHelper.getRequest(), "objectRelationshipId"));
+
+		String externalReferenceCode = null;
+
+		if (_objectEntry != null) {
+			externalReferenceCode = _objectEntry.getExternalReferenceCode();
+		}
+
+		if ((objectRelationship == null) || !objectRelationship.isEdge()) {
+			return _getAPIURL(externalReferenceCode, getObjectDefinition1());
+		}
+
+		String parentObjectEntryAPIURL = _getAPIURL(
+			getParentObjectEntryERC(),
+			_objectDefinitionLocalService.getObjectDefinition(
+				objectRelationship.getObjectDefinitionId1()));
+
+		String apiURL =
+			parentObjectEntryAPIURL + "/" + objectRelationship.getName();
+
+		if (externalReferenceCode != null) {
+			return apiURL + "/" + externalReferenceCode;
```

Comment:
What I meant here was` apiURL += "/" + externalReferenceCode;`
But it's ok to me!